plugins/woocommerce-admin/client/tasks/fills/experimental-products/use-create-product-by-type.ts

```diff
@@ -36,9 +36,12 @@ export const useCreateProductByType = () => {
 				{ _fields: [ 'id' ] }
 			);
 			if ( data && data.id ) {
-				const link = getAdminLink(
+				let link = getAdminLink(
 					`post.php?post=${ data.id }&action=edit&wc_onboarding_active_task=products&tutorial=true`
 				);
+				if ( type === 'physical' ) {
+					link += '&spotlight=true';
```

Comment:
This change is required since it's an entrypoint for an ongoing spotlight experiment. For reference, this [code](https://github.com/woocommerce/woocommerce/blob/fix/re-run-product-task-experiment-v3/plugins/woocommerce-admin/client/tasks/fills/products/product-template-modal.js#L107) is for the control UI